package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -422,7 +422,7 @@ class NCDFReader(base.ReaderBase):
     Current limitations:
 
     * only trajectories with time in ps and lengths in Angstroem are processed
-    * scale_factors are not supported (and not checked)
+    * scale_factors are supported but are not kept for writing
```

Comment:
also support scale_factor for writing

Reply:
As discussed, writing `scale_factor` will be handled in a future request. Extra docs have been written here and in the NCDFWriter to reflect this in 54da919.